tests/playground.html

```diff
@@ -2,12 +2,34 @@
 
 <head>
     <title>Button Counter</title>
+    <script src="http://code.jquery.com/jquery-latest.min.js"></script>
```

Comment:
I think Agam will ask you to create another branch for the playground.html changes. As this branch is for only the detachGlobaEventListeners()

Reply:
Yes, nice catch ;)